### ISSUE TITLE
It is possible to integrate a P5.js canvas/sketch in SwiftUI window (using a webview?)

### ISSUE THREAD

**Issue by jtvargas** [processing/p5.js#6999]
### Topic

Like the title says, I want to know if is possible to integrate a canvas interaction in a Mobile project using Swift/SwiftUI with a Webview or something similar

**Comment by asukaminato0721** [comment]
yes, for example <https://github.com/getflourish/SwiftUIWebP5js>
